data/current/table1-single.csv

```diff
@@ -1,911 +1,920 @@
-"16,848.00","16,871.99",64.77,672.23
-"16,872.00","16,895.99",63.77,671.23
-"16,896.00","16,919.99",62.77,670.23
-"16,920.00","16,943.99",61.77,669.23
-"16,944.00","16,967.99",60.77,668.23
-"16,968.00","16,991.99",59.77,667.23
-"16,992.00","17,015.99",58.77,666.23
-"17,016.00","17,039.99",57.77,665.23
-"17,040.00","17,063.99",56.77,664.23
-"17,064.00","17,087.99",55.77,663.23
-"17,088.00","17,111.99",54.77,662.23
-"17,112.00","17,135.99",53.77,661.23
-"17,136.00","17,159.99",52.77,660.23
-"17,160.00","17,183.99",51.77,659.23
-"17,184.00","17,207.99",50.77,658.23
-"17,208.00","17,231.99",49.77,657.23
-"17,232.00","17,255.99",48.77,656.23
-"17,256.00","17,279.99",47.77,655.23
-"17,280.00","17,303.99",46.77,654.23
-"17,304.00","17,327.99",45.77,653.23
-"17,328.00","17,351.99",44.77,652.23
-"17,352.00","17,375.99",43.77,651.23
-"17,376.00","17,399.99",42.77,650.23
-"17,400.00","17,423.99",41.77,649.23
-"17,424.00","17,447.99",40.77,648.23
-"17,448.00","17,471.99",39.77,647.23
-"17,472.00","17,495.99",38.77,646.23
-"17,496.00","17,519.99",37.77,645.23
-"17,520.00","17,543.99",36.77,644.23
-"17,544.00","17,567.99",35.77,643.23
-"17,568.00","17,591.99",34.77,642.23
-"17,592.00","17,615.99",33.77,641.23
-"17,616.00","17,639.99",32.77,640.23
-"17,640.00","17,663.99",31.77,639.23
-"17,664.00","17,687.99",30.77,638.23
-"17,688.00","17,711.99",29.77,637.23
-"17,712.00","17,735.99",28.77,636.23
-"17,736.00","17,759.99",27.77,635.23
-"17,760.00","17,783.99",26.77,634.23
-"17,784.00","17,807.99",25.77,633.23
-"17,808.00","17,831.99",24.77,632.23
-"17,832.00","17,855.99",23.77,631.23
-"17,856.00","17,879.99",22.77,630.23
-"17,880.00","17,903.99",21.77,629.23
-"17,904.00","17,927.99",20.77,628.23
-"17,928.00","17,951.99",19.77,627.23
-"17,952.00","17,975.99",18.77,626.23
-"17,976.00","17,999.99",17.77,625.23
-"18,000.00","18,023.99",16.77,624.23
-"18,024.00","18,047.99",15.77,623.23
-"18,048.00","18,071.99",14.77,622.23
-"18,072.00","18,095.99",13.77,621.23
-"18,096.00","18,119.99",12.77,620.23
-"18,120.00","18,143.99",11.77,619.23
-"18,144.00","18,167.99",10.77,618.23
-"18,168.00","18,191.99",9.77,617.23
-"18,192.00","18,215.99",8.77,616.23
-"18,216.00","18,239.99",7.77,615.23
-"18,240.00","18,263.99",6.77,614.23
-"18,264.00","18,287.99",5.77,613.23
-"18,288.00","18,311.99",4.77,612.23
-"18,312.00","18,335.99",3.77,611.23
-"18,336.00","18,359.99",2.77,610.23
-"18,360.00","18,383.99",1.77,609.23
-"18,384.00","18,407.99",0.77,608.23
+Amount Effective from January to March 2020/Montant en vigueur en janvier � mars 2020,,,
+Yearly Income (excluding OAS Pension and GIS) From,Yearly Income (excluding OAS Pension and GIS) To,Monthly GIS with Maximum OAS Pension ($),Combined Monthly OAS Pension and GIS ($)
+Revenu annuel (sauf la pension de la SV et le SRG) De,Revenu annuel (sauf la pension de la SV et le SRG) �,SRG mensuel pour montant maximal de la pension de la SV ($),Total mensuel de la pension de la SV et du SRG ($)
+0,23.99,916.38,"1,529.91"

```

Comment:
This looks good, didn't realize GIS rates had gone up again for 2020.

Since the numbers changed, I suspect some tests may need to be maintained, can you run `npm test` and see if any amounts need to be updated?

Reply:
Updated tests